src/Sentry.Unity.Editor/AutoInstrumentation/SentryAssemblyResolver.cs

```diff
@@ -0,0 +1,40 @@
+using System.IO;
+using Mono.Cecil;
+
+namespace Sentry.Unity.Editor
+{
+    public class SentryAssemblyResolver : BaseAssemblyResolver
+    {
+        private readonly string _workingDirectory;
+        private readonly DefaultAssemblyResolver _defaultResolver;
+
+        public SentryAssemblyResolver(string workingDirectory)
+        {
+            _workingDirectory = workingDirectory;
+            _defaultResolver = new DefaultAssemblyResolver();
+        }
+
+        public override AssemblyDefinition Resolve(AssemblyNameReference name)
+        {
+            AssemblyDefinition assembly;
+            try
+            {
+                assembly = _defaultResolver.Resolve(name);
```

Comment:
Would be cleaner as:

```suggestion
                return _defaultResolver.Resolve(name);
```

Same below in the catch.  Then you don't have to declare the variable at all, and it also turns `else throw` into just `throw`.